archimedes/docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
     - /var/run/docker.sock:/var/run/docker.sock:ro
     - archimedes-pycharm-opt:/opt/.pycharm_helpers
     - archimedes-exec:/archimedes-exec
+    - ../vulcan/lib/server/workflows/scripts:/app/vulcan_scripts
```

Comment:
To make it easier to develop scripts in archimedes for vulcan, I mount the vulcan scripts directory as /app/vulcan_scripts.

Reply:
I may be putting my scripts in the wrong place... `vulcan/spec/fixtures/scripts`. Wasn't clear to me where they should go, but the one you reference here organizes within workflows. I picked the one I did because I thought eventually we foresee some scripts being used for more than just one workflow, and by-workflow organization wouldn't make sense then.  @coleshaw ?

Reply:
@dtm2451 let's discuss this tomorrow.

Reply:
Yeah, let's talk about it during Zach's run-through, I think that'll help. Also, @dtm2451 , just as an FYI, the `spec/` directory in our projects houses the Ruby tests, so any scripts or fixtures you find there are used for testing.

No worries where you've put the scripts for now, we can always move things around!